ipatests/test_ipaserver/test_install/test_installutils.py

```diff
@@ -55,21 +55,26 @@ def gpgkey(request, tempdir):
         f.write("allow-preset-passphrase\n")
 
     # daemonize agent (detach from the console and run in the background)
-    subprocess.Popen(
-        [paths.GPG_AGENT, '--batch', '--daemon'],
-        env=env, stdout=devnull, stderr=devnull
+    subprocess.run(
+        ['/usr/bin/systemd-run', '--service-type=forking',
```

Comment:
ALTLinux:
```console
$ which systemd-run
/bin/systemd-run
```

Reply:
So it needs to be part of `ipaplatform`.